modules/appeals_api/spec/support/doc_helpers.rb

```diff
@@ -143,6 +143,10 @@ def self.doc_suffix
     ENV['RSWAG_ENV'] == 'dev' ? '_dev' : ''
   end
 
+  def self.doc_url_prefix
+    ENV['RSWAG_ENV'] == 'dev' ? 'dev-' : ''
+  end
```

Comment:
This is very similar to the `DocHelpers.doc_suffix` above. I created it so that it could be used in `rswag_config.rb` to provide dev developer portal links when the environment is dev or staging.